site/app/views/admin/AdminGradeableView.php

```diff
@@ -792,7 +851,7 @@ function display_graders($graders, $have_old, $g_grade_by_registration, $graders
 </div>
 
 <link rel="stylesheet" href="http://code.jquery.com/ui/1.10.2/themes/smoothness/jquery-ui.css" />
-<link type="text/css" rel="stylesheet" href="css/jquery-ui-timepicker-addon.css" />
+<link type='text/css' rel='stylesheet' href="http://trentrichardson.com/examples/timepicker/jquery-ui-timepicker-addon.css" />
```

Comment:
No, add this css locally. We should only ever trust on relying on external sources for things that are a) well-known and established and can trust it'll probably never go down (ex: font-awesome) and b) is served via a CDN (ex: font-awesome). This does not fit either of those criteria.